lib/rules/multiline-ternary.js

```diff
@@ -67,7 +72,16 @@ module.exports = {
                                 start: firstTokenOfTest.loc.start,
                                 end: lastTokenOfTest.loc.end
                             },
-                            messageId: "unexpectedTestCons"
+                            messageId: "unexpectedTestCons",
+                            fix: fixer => (hasComments ? null : (
+                                fixer.replaceTextRange(
+                                    [
+                                        lastTokenOfTest.range[1],
+                                        firstTokenOfConsequent.range[0]
+                                    ],
+                                    " ? "
+                                )
+                            ))
```

Comment:
Similar to the other comment, this could add extra spaces, for example:

```js
a
?b
:c
```

will be autofixed to:

```js
a ? b : c
```

Thoughts about just removing range between `lastTokenOfTest` and `questionToken` if these two tokens are not on the same line, and the same for `questionToken` and `firstTokenOfConsequent` if they're not on the same line? Then, `space-infix-ops` could add spaces if it was enabled.

(same question for `messageId: "unexpectedConsAlt"`)